### ISSUE TITLE
[Express] Session id don't match

### ISSUE THREAD

**Issue by riderodd** [Shopify/shopify-app-node#732]
Hi Shopify team,

I'm using the new express version to develop a Shopify app. It's working great, except when I make graphql query.

# Issue summary

When I do a graphql from client side request, the server returns a HTTP 403 and a starts a redirection loop.

## Expected behavior

I should be able to query server graphql proxy being authenticated.

## Actual behavior

The graphql requests returns a 403, making a redirection on auth. After 5 or 6 redirects, Shopify admin says:
![image](https://user-images.githubusercontent.com/2619590/157859897-b130a1ed-2e73-4032-9dde-b1cfca1c593d.png)

## Steps to reproduce the problem

1. `npx degit shopify/shopify-app-node#sample_app_updates`
2. `npm i`
3. `shopify app serve`
4. Follow the CLI link, install the app, and try to open it in a browser
5. Get redirected, and then see the Shopify error message as shown above

## Reduced test case

Juste use the sample_app_updates branch

## Specifications

- Browser: Chrome
- Operating System: Windows 11, running app from WSL
- @shopify/shopify-api version : ^3.0.0-rc.1

## First thoughts
I've identified that session IDs differs on Shopify and Server:
Added this line in server/middleware/verify-request.js:13
`console.log(Object.keys(Shopify.Context.SESSION_STORAGE.sessions))`
Gave me:
`[ 'app-demo.myshopify.com_79460237504' ]`

And added a log too in ShopifyOAuth.getCurrentSessionId (/node_modules/@shopify/shopify-api/dist/utils/load-current-session.js:23)
`console.log(sessionId);`
Gave me:
`app-demo.myshopify.com_78758609088`

As session IDs differs, verifyRequest() thinks that the request cannot be authenticated, and returns a HTTP 403. I don't understand why those ids differs.

Thanks for the help 😊

**Comment by Michael-Gibbons** [comment]
Interesting, I believe the issue is not installing the app before trying to degit. The `shopify node app create` command works by creating an app admin side, then cloning the current main branch. This generates a `shopify-cli.yml` file which contains your organization id and project type. 

This issue should be fixed once the sample app updates branch is merged into master, but until then, I created and installed an app using the shopify-cli normally, then the nextjs build is cloned into my directory, verified the installation worked, THEN I deleted everything in the directory except for the `shopify-cli.yml` file, and used the degit command, and it seems to be working correctly for me.

tldr, install app with cli before replacing contents with degit
